README.md

```diff
@@ -25,3 +25,18 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+By ignoring the constant factor(splitting array=O(1), combining results=O(1)), the $T(n)=3T(\frac{n}{3})$, becuase the array devided into
```

Comment:
We cannot simply ignore the constant part in the recurrence relation. Maybe the summation of those has some property that needs to be considered.